src/main/java/frc/robot/Constants.java

```diff
@@ -31,12 +33,8 @@ public class Constants {
     public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(204.256);
     public static final MechanicalConfiguration MODULE_CONFIGURATION = SdsModuleConfigurations.MK4I_L2;
     public static final String CANIVORE_BUS_NAME = "CANivore Bus 1";
-    public static final double LIMELIGHT_FORWARD_OFFSET = 0.172; //limelight x offset from center of robot
-    public static final double LIMELIGHT_SIDE_OFFSET = -0.325;
-    public static final double LIMELIGHT_UP_OFFSET = 0.197;
-    public static final double LIMELIGHT_YAW_OFFSET = -40; //make sure you REALLY have this right...
-    public static final double LIMELIGHT_PITCH_OFFSET = -22.0;
-    public static final double LIMELIGHT_ROLL_OFFSET = 4.0;
+
+    public static final Pose3d LIMELIGHT_OFFSETS = new Pose3d(0.172, -0.325, 0.197, new Rotation3d(Math.toRadians(10.0), Math.toRadians(-23.0), Math.toRadians(-46.0)));
```

Comment:
the to/from radians is going to get confusing, if it isn't already. You can do this constructor https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/geometry/Rotation3d.html#%3Cinit%3E(edu.wpi.first.units.measure.Angle,edu.wpi.first.units.measure.Angle,edu.wpi.first.units.measure.Angle), and do `Angle.ofBaseUnits(10.0, Units.Degrees)` to keep it clear